packages/docusaurus/src/commands/start.ts

```diff
@@ -101,6 +101,7 @@ export default async function start(
         inject: false,
         filename: 'index.html',
         title: siteConfig.title,
+        noIndex: siteConfig.noIndex,
```

Comment:
@slorber is this really necessary when developing website? I mean, `noIndex` is relevant only in production build.

Reply:
agree, this seems unnecessary we only need noindex for build, as a site config (not a cli option)

Reply:
Either way it is the same to me. I just need to add this option like we used to have in V1.